tasks/run-file-updates/run-file-updates.yaml

```diff
@@ -105,7 +105,7 @@ spec:
 
           results=$(jq -r '.status.results' <<< "${IRjson}")
           internalRequestPipelineRunName="$(jq -jr '.internalRequestPipelineRunName // ""' <<< "${results}")"
-          internalRequestTaskRunName="$(jq -jr '.internalRequestTaskRunName / ""' <<< "${results}")"
+          internalRequestTaskRunName="$(jq -jr '.internalRequestTaskRunName // ""' <<< "${results}")"
```

Comment:
It seems these results are not working. Any idea why @scoheb ? I see they are set here: https://github.com/konflux-ci/release-service-catalog/blob/development/internal/pipelines/process-file-updates/process-file-updates.yaml

Reply:
We got to the bottom of this - see this jira comment: https://issues.redhat.com/browse/RELEASE-1304?focusedId=26122793&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-26122793

Anyway, it doesn't block this fix.